spec/features/families/admin_views_all_active_families_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+feature "Admin can see all the active families when in families#index" do
```

Comment:
A suggestion to change the description to be: "Admin sees active and inactive families"